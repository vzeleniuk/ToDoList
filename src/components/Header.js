import React from 'react';
import {Link} from 'react-router-dom';

export const Header = () => {
  return(
    <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
      <div className="container">
        <Link to={"/"} className="navbar-brand">My TODO List</Link>
        <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
          <span className="navbar-toggler-icon"></span>
        </button>
        <div className="collapse navbar-collapse" id="navbarCollapse">
          <ul className="navbar-nav ml-auto">
            <li><Link to={"/"} className="nav-link">Root</Link></li>
            <li><Link to={"/user"} className="nav-link">User</Link></li>
          </ul>
        </div>
      </div>
    </nav>
  );
};