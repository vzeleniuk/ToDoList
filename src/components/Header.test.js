import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Header from './Header'

describe('<Header />', () => {
  test('Header should render as expected', () => {
    const wrapper = shallow(<Header />);
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  })
})