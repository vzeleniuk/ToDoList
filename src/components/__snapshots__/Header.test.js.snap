// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Header should render as expected 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-dark"
>
  <div
    className="container"
  >
    <Link
      className="navbar-brand"
      replace={false}
      to="/"
    >
      My TODO List
    </Link>
    <div
      className="collapse navbar-collapse"
      id="navbarCollapse"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <li>
          <Link
            className="nav-link"
            replace={false}
            to="/"
          >
            Home
          </Link>
        </li>
        <li>
          <Link
            className="nav-link"
            replace={false}
            to="/user"
          >
            User
          </Link>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
